src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
+        super(color);
+        this.height = height;
+        this.bothside = bothside;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Right Triangle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Both side length: " + bothside);
+        System.out.println("Height length: " + height);
+        System.out.println("Area: " + getArea());
```

Comment:
Don't put all behavior into a single method if the operations are conceptually different from each other. In this case, consider separating the logic for drawing the figure from the logic for printing its properties.